Automattic/vip-go-mu-plugins
# Show plugin update notices on the plugins page.

Core restricts showing plugin update notices to users with update_plugins, which is not granted to anyone on VIP.

Instead, copy core's wp_plugin_update_rows() logic and use the manage_options cap instead.

There are other checks further down the stack to prevent users without the update_plugins cap to actually trigger updates.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Go to `wp-admin/plugins.php`
1. Install an old version of a plugin (e.g. goto https://en-ca.wordpress.org/plugins/edit-flow/advanced/ -- then scroll to bottom and download an old version)
1. Reload plugins page.

The newly added plugin should now have a notice saying a new version is available.
